staging/src/k8s.io/client-go/pkg/version/base.go

```diff
@@ -43,7 +43,7 @@ var (
 	gitMinor string = "" // minor version, numeric possibly followed by "+"
 
 	// semantic version, derived by build scripts (see
-	// https://git.k8s.io/community/contributors/design-proposals/release/versioning.md
```

Comment:
I think this should link to one of the following rather than an archived doc:
* https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md#kubernetes-release-versioning
* https://kubernetes.io/releases/version-skew-policy/